File: AN_2/SEM_1/MAP/lab2/src/container/StackContainer.java
package container;

import model.Task;

public class StackContainer extends AbstractContainer
{
    public StackContainer()
    {
        taskuri = new Task[10];
        size = 0;
    }

    @Override
    public Task remove()
    {
        if (!isEmpty())
        {
            Task task = taskuri[--size];
            taskuri[size] = null;
            return task;
        }
        return null;
    }
}

File: AN_2/SEM_1/MAP/lab9/lab9/Container/StackContainer.cs
using Model;

namespace Container
{
    public class StackContainer : AbstractContainer
    {
        public StackContainer()
        {
            taskuri = new Tasks[10];
            size = 0;
        }

        public override Tasks Remove()
        {
            if (!IsEmpty())
            {
                Tasks task = taskuri[--size];
                taskuri[size] = null;
                return task;
            }
            return null;
        }
    }
}
